### DISCUSSION TITLE
Line numbers in Code Blocks

### DISCUSSION THREAD

**Discussion by bikrantlabs** [forem/forem#19557] [Feature Requests]
I am writing blogs related to programming and I've also been reading a lot on them. It would be better if line numbers were added on code block (I could insert an image from text editor, but other people won't be able to copy it) and it would be easy to mention specific line number to explain what the part of the code is actually doing.

**Comment by wescpy** [comment]
ReType has a simple syntax on enabling line numbering in their code samples: <https://retype.com/components/code-block/#line-numbers> ... it would be great to adopt the same.
